examples/next/pages/ui/components/geo/standalone-location-search/index.page.tsx

```diff
@@ -12,6 +12,39 @@ const SAN_FRANCISCO = {
   longitude: -122.431,
 };
 
+const onLocationSearchResult = (event) => {
+  console.log('Result event is firing');
+  console.log(event);
+};
+
+const onLocationSearchResults = (event) => {
+  console.log('Results event is firing');
+  console.log(event);
+};
+
+const onLocationSearchClear = () => {
```

Comment:
Does the `clear` listener not emit an `event` argument?

Reply:
I don't beleive so, but will take another look

Reply:
It does not